package.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "tezster",
-  "version": "0.2.3",
+  "version": "0.2.4",
   "description": "A JavaScript-based private Tezos Blockchain Dev Tool that lives entirely on your local machine.",
   "main": "tezster.js",
   "scripts": {
+    "postinstall": "mkdir -p /tmp/tezster && cp -n ./config.json /tmp/tezster/config.json",
```

Comment:
this is better way of setting up config.json, but this will overwrite existing file.

Reply:
no that will not overwrite, we are using -n flag to avoid overwriting the existing file. 

the above script will run perfect for linux and mac. For windows "copy" is supported to copy instead of "cp" command. I was thinking to add separate script as "postinstall-win" for windows specially and will update the docs accordingly. Should I go forward?